src/App.tsx

```diff
@@ -15,71 +15,69 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export const App: React.FC = () => {
+  const [chosenElement, chosenGood] = useState(goods[8]);
+  const [reset, onReset] = useState(true);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const clear = () => {
+    chosenGood(goods[-1]);
```

Comment:
```suggestion
    chosenGood("");
```
Just set an empty string